### DISCUSSION TITLE
tutorials/intro-strawberry/03-strawberry

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#116] [course feedback]
# tutorials/intro-strawberry/03-strawberry

Learn about Strawberry, a Python library for building GraphQL APIs using type annotations.

https://www.apollographql.com/tutorials/intro-strawberry/03-strawberry

**Comment by ghost** [comment]
Really interested to the ariadne

**Reply by mabuyo** [reply]
Thanks for letting us know @williamzborja !

**Comment by robwma** [comment]
Minor issue, clicking Next does not give me a green checkbox even though I went through this page

**Reply by mabuyo** [reply]
👋🏽 Good callout @robwma! The `-` beside the lesson in the navigation indicates that there are no tasks or quizzes to be completed, so it will always stay as `-`, but I can see how that can be confusing!

**Comment by bparanj** [comment]
Why are we using FastAPI here instead of using only Strawberry?

**Reply by mabuyo** [reply]
Hi there! 👋🏽 It's not required to use FastAPI, but Strawberry does support integrations with many popular HTTP web frameworks (including [FastAPI](https://strawberry.rocks/docs/integrations/fastapi), [Flask](https://strawberry.rocks/docs/integrations/flask) and [Django](https://strawberry.rocks/docs/integrations/django)). You can use any or none! 

For consistency with our other introductory courses in other programming languages, we wanted to easily use Apollo Sandbox as the GraphQL IDE, which you can set using the `graphql_ide` option through the FastAPI integration.
